autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/ui/credential/management/AutofillManagementRecyclerAdapter.kt

```diff
@@ -67,7 +68,7 @@ class AutofillManagementRecyclerAdapter(
         val item = listItems[position] as ListItem.Credential
         with(viewHolder.binding) {
             username.text = item.credentials.username
-            domain.text = item.credentials.domain
+            domain.text = titleExtractor.extract(item.credentials)
```

Comment:
NIT: It would be less confusing to name the fields as title and subtitle since at least the title doesn't always refer to domain anymore.